todo_app/flask_config.py

```diff
@@ -4,5 +4,12 @@
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
+
+    TRELLO_API_KEY=
+    TRELLO_API_SECRET=
+    TRELLO_BOARD_ID=
```

Comment:
These variables should be reading in the values from environment variables like SECRET_KEY does - here it looks like you're expecting developers to manually add the values here